content/articles/reactivate-subscription.md

```diff
@@ -9,7 +9,7 @@ categories:
 
 When you unsubscribe, or your account was [closed due to non-payment](/articles/account-suspended/), all of your domains and records remain in your account. You can reactivate your subscription and start using DNSimple's services again at any time.
 
-When your account is deactivated, [WHOIS privacy](/articles/whois-privacy/) and [domain auto-renewal](/articles/domain-auto-renewal/) are immediately disabled. If you want to use these features, you'll need to manually re-activate them.
+When your account is deactivated, [WHOIS privacy](/articles/whois-privacy/), [domain auto-renewal](/articles/domain-auto-renewal/), and [email forwarding](/articles/email-forwarding/) are immediately disabled. If you want to use these features, you'll need to manually re-activate them.
```

Comment:
This  part here is incorrect:

> If you want to use these features, you'll need to manually re-activate them.

When an account is resubscribed, it retains the previous preferences., which means that any domains with auto-renewal enabled will have auto-renewal reactivated automatically, and the same applies to email forwarding settings.

Reply:
@Pemacaa could we say this? 

> If you reactivate your account, all previous preferences will be reactivated as well, including WHOIS privacy, auto-renewal, and email forwarding.

Reply:
@itsalyse that works for me.

Reply:
Thank you for handling this!